### ISSUE TITLE
Close POAM V2-CM-6 ("It is possible to view the directory listing")

### ISSUE THREAD

**Issue by dlapiduz** [18F/cg-product#394]
In order to demonstrate that our system has an acceptable level of risk, we need to close this identified potential risk as resolved or invalid.

**Implementation sketch**

See [POAM](https://docs.google.com/spreadsheets/d/16igVl8cD3SqeX5_SOn5Su34KmwMRnP20gPbfQlqIwfM/edit#gid=1701775784) for details, under V2-CM-6.

The described issue is:
* Look at scans for https://logs.fr.cloud.gov/ (specific URLs are in scan)
* Disable directory browsing. If this is required, make sure the listed files does not induce risks.

Todo items:
- [ ] Our 18F AO has documented rationale for accepting this as a risk. Related GitHub issue with technical details is at https://github.com/18F/cg-atlas/issues/129 ("Update Kibana to stop exposing directory listings")
- [ ] We've asked 3PAO to verify this.

**Acceptance Criteria**:
- [ ] Our 3PAO has verified this.

**Comment by mogul** [comment]
We're waiting on an upstream release of Kibana to address this. The release candidate has just been tagged in the last week: https://github.com/elastic/kibana/releases/tag/v5.0.0-rc1

**Comment by mogul** [comment]
Refer to https://github.com/18F/cg-atlas/issues/129 for state of remediation.

**Comment by brittag** [comment]
This is now a duplicate of https://github.com/18F/cg-product/issues/414 since it's an accepting-risk item instead of a technical remediations item.
